Package.swift

```diff
@@ -0,0 +1,21 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+  name: "PubNubChat",
+  platforms: [.iOS(.v14)],
+  products: [
+    .library(
+      name: "PubNubChat",
+      targets: ["PubNubChatRemoteBinaryPackage"]),
+  ],
+  targets: [
+    .binaryTarget(
+      name: "PubNubChatRemoteBinaryPackage",
+      url: "https://github.com/pubnub/kmp-chat/releases/download/1.0.0/PubNubChat.xcframework.zip",
+      checksum: ""
```

Comment:
I'll provide final checksum if we finish implementing `kmp-chat` and merge all PRs